fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1177,6 +1177,7 @@ public void finishTransaction(long transactionId, Set<Long> errorReplicaIds) thr
                     transactionState.clearErrorMsg();
                     transactionState.setTransactionStatus(TransactionStatus.VISIBLE);
                     unprotectUpsertTransactionState(transactionState, false);
+                    transactionState.resetTabletCommitInfos();
```

Comment:
You can reset it now, after that in `applyVisibleLog`, we still need `tabletCommitInfos`, you can move it to `applyVisibleLog`.

Reply:
tks reveiw.